0.wave1_data/3.process_features/extract_sc_interstellar.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process single cell morphology features for CellProfiler readouts
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+
+import pycytominer
```

Comment:
Consider removing this import for pycytominer (the `from`-based imports below appear to be what are implemented).
```suggestion
```

Reply:
Good catch! It is removed.